src/CommunityToolkit.Maui/Views/Expander/Expander.shared.cs

```diff
@@ -66,14 +67,23 @@ public Expander()
 	}
 
 	/// <summary>
-	///	Triggered when 
+	///	Triggered when the value of <see cref="IsExpanded"/> changes
 	/// </summary>
 	public event EventHandler<ExpandedChangedEventArgs> ExpandedChanged
 	{
 		add => tappedEventManager.AddEventHandler(value);
 		remove => tappedEventManager.RemoveEventHandler(value);
 	}
 
+	/// <summary>
+	/// The Action that fires when <see cref="Header"/> is tapped.
+	/// By default, this <see cref="Action"/> runs <see cref="ResizeExpanderInItemsView(TappedEventArgs)"/>.
+	/// </summary>
+	/// <remarks>
+	/// Warning: Overriding this <see cref="Action"/> may cause <see cref="Expander"/> to work improperly when placed inside of a <see cref="CollectionView"/> and placed inside of a <see cref="ListView"/>.
+	/// </remarks>
+	public Action<TappedEventArgs>? HandleHeaderTapped { get; set; }
```

Comment:
this should be static or the dev will need to implement it for each instance of `Expander`
see 
- https://github.com/CommunityToolkit/Maui/blob/58240c8109f8d1bfe8313ddb4dc81898671ee728/src/CommunityToolkit.Maui/Views/Expander/Expander.shared.cs#L255

Reply:
True, but I think it's best to keep it non-static. 

If a dev is using multiple `Expander`s in their app, they may not want to override the behavior the same way for each. 

And if a dev_does want to implement the same behavior for every `Expander`, they can subclass it like so:

```cs
public class CustomExpander : Expander
{
  public CustomExpander()
  {
    this.HandleHeaderTapped = CustomHeaderTappedMethod;
  }
  
  void CustomHeaderTappedMethod(TappedEventArgs args)
  {
    // ...
  }
}
```

Reply:
@brminnick  Can you double-check if the sample is still working? Since we use a collection, I believe we will create one per cell.

Reply:
Of course! 

Yup, here is `Expander` inside of a `CollectionView`, working on iOS + MacCatalyst:
<img width="1046" alt="image" src="https://user-images.githubusercontent.com/13558917/212440894-285d087d-0b82-4383-af8b-a617f09fd59c.png">